### PR TITLE
AddSignatory mutation

### PR DESCRIPTION
**Pull request by mickhansen**
I considered whether we should return a client aware SignatureOrder type, instead of just returning the converted GraphQL type, but for now i'll be sticking to just top level arguments and the implementer will have to carry the return values/ids around

### CODE REVIEW

File: Criipto.Signatures/Operations.cs [hunk 1]
```diff
@@ -48,6 +48,40 @@ public class CreateSignatureOrderMutation {
...
+      /// </summary>
+      public static GraphQLRequest Request(object variables = null) {
+        return new GraphQLRequest {
+          Query = AddSignatoryDocument,
+          OperationName = "addSignatory",
+          Variables = variables
+        };
+      }
+
+      /// <remarks>This method is obsolete. Use Request instead.</remarks>
```
Comment (sgryt) [inline Criipto.Signatures/Operations.cs:65]: Why introduce an Obsolete method in the first place ?

Comment (mickhansen) [reply Criipto.Signatures/Operations.cs:65]: @sgryt No clue, the codegen really isn't that great for C#

**Comment by sgryt** [comment]
https://github.com/criipto/criipto-signatures-dotnet/actions/runs/4376265214/jobs/7658121349

100 sec timeout for NuGet access seems excessive ?

**Comment by mickhansen** [comment]
> https://github.com/criipto/criipto-signatures-dotnet/actions/runs/4376265214/jobs/7658121349
> 
> 100 sec timeout for NuGet access seems excessive ?

Agree, but no clue where i'd begin to configure it, don't think failures happen often.

**Pull request merged**